js/app/test/outbreak_forecast.spec.ts

```diff
@@ -21,24 +15,12 @@ test("matplotlib", async ({ page }) => {
-			}
-		]
-	]);
-	await mock_theme(page);
-	await page.goto("http://localhost:9876");
-
 	await page.getByLabel("Plot Type").click();
-	await page.getByRole("button", { name: "Matplotlib" }).click();
+	await page.getByRole("button", { name: "Plotly" }).click();
```

Comment:
This was always wrong since we were mocking it really didn't matter what we selected.